expression/builtin_encryption_vec_test.go

```diff
@@ -22,13 +22,15 @@ import (
 )
 
 var vecBuiltinEncryptionCases = map[string][]vecExprBenchCase{
-	ast.AesEncrypt:         {},
-	ast.Uncompress:         {},
-	ast.AesDecrypt:         {},
-	ast.Compress:           {},
-	ast.MD5:                {},
-	ast.SHA:                {},
-	ast.RandomBytes:        {},
+	ast.AesEncrypt: {},
+	ast.Uncompress: {},
+	ast.AesDecrypt: {},
+	ast.Compress:   {},
+	ast.MD5:        {},
+	ast.SHA:        {},
+	ast.RandomBytes: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETInt}, geners: []dataGenerator{&rangeInt64Gener{1, 1024}}},
+	},
```

Comment:
The reason causing unit test problems is that our testing framework compares results of vectorized functions and row-based functions to check if there is any bug.
For builtin functions like `Rand`, `RandBytes`, we can't use this way to test them.

So I think you can remove these lines.

Reply:
ok, I will fix it, and resolve conflicts.